logilica_weekly_report/configuration_schema.py

```diff
@@ -42,6 +42,10 @@
                         "type": "array",
                         "items": {"type": "string"},
                     },
+                    "membership_boards": {
+                        "type": "array",
+                        "items": {"type": "string"},
+                    },
```

Comment:
Should any of the test scenarios in [`test_configuration_schema.py`](https://github.com/developerproductivity/logilica-weekly-report/blob/main/tests/test_configuration_schema.py) be tweaked for this change?

Reply:
I have updated that now, :-)